source/adiabatic_conditions/initial_profile.cc

```diff
@@ -72,11 +72,14 @@ namespace aspect
           in.position[0] = representative_point;
           in.temperature[0] = temperatures[i-1];
           in.pressure[0] = pressures[i-1];
+          in.velocity[0] = Tensor <1,dim> ();
 
           for (unsigned int c=0; c<n_compositional_fields; ++c)
             in.composition[0][c] = this->get_compositional_initial_conditions().initial_composition(representative_point, c);
 
           in.strain_rate.resize(0); // we do not need the viscosity
+          in.cell = this->get_dof_handler().end(); // and do not know the cell index
```

Comment:
I can use that, but how would I test in the material model, if the in.cell is set? Can I check for `in.cell != typename DoFHandler<dim>::active_cell_iterator()` ?
Currently I do: `in.cell != this->get_dof_handler().end()`